src/cowpatch/text_elements.py

```diff
@@ -290,27 +476,251 @@ def _create_svg_object(self, width=None, height=None):
+            with the svg...)
+        dpi : int or float
+            dots per square inch, default is 96 (standard)
+        verbose : bool
+            If ``True``, print the saving information. The package default
+            is defined by cowpatch's own rcParams (the base default is
+            ``True``), which is used if verbose is ``None``. See Notes.
+
+        Notes
+        -----
+        If width and/or height is None, the approach will attempt to define
+        acceptable width and height.
+
+        The ``verbose`` parameter can be changed either directly with defining
+        ``verbose`` input parameter or changing
+        ``cow.rcParams["show_verbose"]``.
+
+        If run from the command line, this approach leverage matplotlib's
+        plot render to show a static png version of the image. If run inside
+        a jupyter notebook, this approache presents the actual svg
+        representation.
+
+        See also
+        --------
+        cow.patch.show : same function but for a cow.patch object
         """
+        # updating width and height if necessary (some combine is none)
+        width, height = self._default_size(width=width, height=height)
+
+        # global default for verbose (if not provided by the user)
+        if verbose is None:
+            verbose = rcParams["show_verbose"]
+
+        svg_obj, (actual_width_pt, actual_height_pt) = \
+            self._svg(width_pt=from_inches(width, "pt", dpi=dpi),
+                       height_pt=from_inches(height, "pt", dpi=dpi))
+
+        _show_image(svg_obj,
+                    width=to_inches(actual_width_pt, "pt", dpi=dpi),
+                    height=to_inches(actual_height_pt, "pt", dpi=dpi),
+                    dpi=dpi,
+                    verbose=verbose)
+
+    def __str__(self):
+        self.show()
+        return "<text (%d)>" % self.__hash__()
+
+    def __repr__(self):
+        out = "label:\n" +  "  |" + re.sub("\n", "\n  |", self.label) +\
+              "\nelement_text:\n  |" + self.element_text.__repr__() +\
+              "\n_type:\n  |\"" + self._type +"\""
+
+        return "<text (%d)>" % self.__hash__() + "\n" + out
+
+    def __hash__(self):
+        return hash(tuple(self.__dict__))
+
+class cow_title(themeable):
+    """
+    text class for cow.patch titles
+
+    Parameters
+    ----------
+    theme_element : element_text
+    """
+    pass
+
+class cow_subtitle(themeable):
+    """
+    text class for cow.patch subtitles
+
+    Parameters
+    ----------
+    theme_element : element_text
+    """
+    pass
+
+class cow_caption(themeable):
+    """
+    text class for cow.patch captions
+
+    Parameters
+    ----------
+    theme_element : element_text
+    """
+    pass
+
+class cow_tag(themeable):
+    """
+    text class for cow.patch tags
+
+    Parameters
+    ----------
+    theme_element : element_text
+    """
+    pass
+
+class cow_text(cow_title, cow_subtitle, cow_caption, cow_tag):
```

Comment:
`plotnine` `themeables` have a weird inverse class structure - this indicates that `cow_title`, etc. inherits from `cow_text`.